File: 151-reverse-words-in-a-string/reverse-words-in-a-string.java
class Solution {
    public String reverseWords(String s) {
        String[] splited_string=s.trim().split("\\s+");
        ArrayDeque <String> stack=new ArrayDeque<>();
        for(String temp:splited_string){
            stack.push(temp);
        }
        return String.join(" ",stack);
    }
}

File: 151-reverse-words-in-a-string/reverse-words-in-a-string.py
class Solution:
    def reverseWords(self, s: str) -> str:
        s = s.strip()
        while s.find("  ") >= 0:
            s = s.replace("  ", " ")
        reversed_list = s.split(" ")
        reversed_list.reverse()
        return " ".join(reversed_list)
